cppcache/src/ClientProxyMembershipID.hpp

```diff
@@ -51,26 +46,37 @@ class ClientProxyMembershipID : public DSMemberForVersionStamp {
                           const std::chrono::seconds durableClientTimeOut =
                               std::chrono::seconds::zero());
 
-  // This constructor is only for testing and should not be used for any
-  // other purpose. See testEntriesMapForVersioning.cpp for more details
+  /**
+   * This constructor is only for testing and should not be used for any
+   * other purpose. See testEntriesMapForVersioning.cpp for more details
+   */
   ClientProxyMembershipID(const uint8_t* hostAddr, uint32_t hostAddrLen,
                           uint32_t hostPort, const char* dsname,
                           const char* uniqueTag, uint32_t vmViewId);
-  // ClientProxyMembershipID(const char *durableClientId = nullptr, const
-  // uint32_t durableClntTimeOut = 0);
+
   ClientProxyMembershipID();
+
   ~ClientProxyMembershipID() noexcept override;
+
   static void increaseSynchCounter();
+
   static std::shared_ptr<Serializable> createDeserializable() {
     return std::make_shared<ClientProxyMembershipID>();
   }
-  // Do an empty check on the returned value. Only use after handshake is done.
+
+  /**
+   * Do an empty check on the returned value. Only use after handshake is done.
+   */
```

Comment:
I'm not sure this comment adds any value.  Also the name of this function doesn't appear to describe it correctly - it returns member var `clientId_`, maybe just call it getClientId?

Reply:
I didn't want to get that deep into refactoring this. I wanted to keep to protocol. I just hate seeing documentation comments using the wrong comment format. We could wack the whole comment. I will leave the refactoring of the name and purpose up to someone else.

Reply:
Ok, I am going to fix it. I didn't even look at the implementation previously. You are right, it is just a getter.